omc3/model/accelerators/lhc.py

```diff
@@ -486,12 +488,48 @@ def _get_madx_main_sequence_loading(self) -> str:
                 "has to be specified (--lhcmode option missing?)."
             )
 
-    def get_update_correction_script(self, outpath: Path | str, corr_files: Sequence[Path | str]) -> str:
+    def get_update_correction_script(self, outpath: Path | str, corr_files: Sequence[Path | str], update_dpp: bool = False) -> str:
```

Comment:
Surely `update_dpp` here isn't expected to be a bool, considering the later checks and value passed to a function which expects `float | str`.

Reply:
It should be boolean, I should have never changed it to a float.